### ISSUE TITLE
CANNOT USE SCENE

### ISSUE THREAD

**Issue by devins-07** [bwp91/homebridge-govee#94]
hello, i was in the process of setting this up and i was setting up scenes and i cant get the "[Govee] [Device] the AWS code for this scene/mode is shown on the next line(s):
MwUECAAAAAAAAAAAAAAAAAAAADo=" part to show up. i was following the instructions here https://github.com/bwp91/homebridge-govee/wiki/Scene%2C-Music%2C-DIY-Modes
with no luck. here is part of the log:

8/3/2021, 7:13:15 PMapple starting
8/3/2021, 7:13:18 PMappleLoading 1 platforms...
8/3/2021, 7:13:18 PMappleGoveeInitialising plugin v4.0.2 | Node v14.17.4 | HB v1.3.4 | HOOBS v4...
8/3/2021, 7:13:18 PMappleGoveePlugin initialised. Setting up accessories....
8/3/2021, 7:13:18 PMappleBridge is running on port 51826.
8/3/2021, 7:13:19 PMappleGovee[Indoor Thermometer] initialising with options {"logging":"debug","lowBattThreshold":20}.
8/3/2021, 7:13:19 PMappleGovee[Indoor Thermometer] initialised with id [16:10:A4:C1:38:10:F6:1D].
8/3/2021, 7:13:19 PMappleGovee[Door lights] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"yes","enableBT":"no","logging":"debug"}.
8/3/2021, 7:13:19 PMappleGovee[Door lights] initialised with id [3B:BF:A4:C1:38:CE:75:63].
8/3/2021, 7:13:19 PMappleGovee[Window lights] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"yes","enableBT":"no","logging":"debug"}.
8/3/2021, 7:13:19 PMappleGovee[Window lights] initialised with id [29:B6:A4:C1:38:A6:4D:E5].
8/3/2021, 7:13:19 PMappleGovee[Desk lights] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"yes","enableBT":"no","logging":"debug"}.
8/3/2021, 7:13:19 PMappleGovee[Desk lights] initialised with id [3E:57:A4:C1:38:16:0C:73].
8/3/2021, 7:13:19 PMappleGoveeWARNING[Bed lights] is not currently supported, do create a Github issue with the following info:
8/3/2021, 7:13:19 PMappleGoveeWARNING[object Object]
8/3/2021, 7:13:19 PMappleGovee✓ Setup complete. Want to see this plugin in your own language? Let me know!
8/3/2021, 7:13:19 PMappleGovee[noble] stateChange: poweredOn.
8/3/2021, 7:13:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:13:19 PMappleGoveeWARNING[Door lights] has been reported [offline].
8/3/2021, 7:13:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:13:19 PMappleGoveeWARNING[Indoor Thermometer] has been reported [offline].
8/3/2021, 7:13:19 PMappleGovee[Indoor Thermometer] current battery [67%].
8/3/2021, 7:13:19 PMappleGovee[Indoor Thermometer] current temperature [26.1°C].
8/3/2021, 7:13:19 PMappleGovee[Indoor Thermometer] current humidity [41.1%].
8/3/2021, 7:13:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:13:19 PMappleGovee[Window lights] has been reported [online].
8/3/2021, 7:13:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:13:19 PMappleGovee[Desk lights] has been reported [online].
8/3/2021, 7:13:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:13:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:13:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:13:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:13:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:13:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:13:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:13:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:14:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:14:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:14:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:14:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:14:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:14:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:14:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:14:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:14:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:14:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:14:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:14:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:14:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:14:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:14:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:14:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:15:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:15:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:15:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:15:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:15:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:15:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:15:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:15:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:15:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:15:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:15:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:15:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:15:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:15:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:15:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:15:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:16:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:16:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:16:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:16:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:16:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:16:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:16:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:16:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:16:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:16:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:16:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:16:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:16:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:16:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:16:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:16:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:17:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:17:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:17:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:17:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:17:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:17:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:17:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:17:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:17:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:17:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:17:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:17:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:17:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:17:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:17:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:17:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:18:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:18:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:18:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:18:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:18:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:18:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:18:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:18:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:18:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:18:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:18:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:18:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:18:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:18:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:18:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:18:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:19:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:19:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:19:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:19:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:19:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:19:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:19:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:19:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:19:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:19:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:19:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:19:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:19:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:19:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:19:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:19:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:20:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:20:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:20:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:20:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:20:19 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:20:19 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:20:19 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:20:19 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:20:34 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:20:34 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:20:34 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:20:34 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:20:49 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:20:49 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/3/2021, 7:20:49 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:20:49 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:21:04 PMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/3/2021, 7:21:04 PMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/3/2021, 7:21:04 PMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/3/2021, 7:21:04 PMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].

**Comment by bwp91** [comment]
Hi @devins-07 

You have `"disableAWS":"yes"` in the configuration for your lights - you need to untick this box in the settings and try again!

**Comment by devins-07** [comment]
> Hi @devins-07
> 
> You have `"disableAWS":"yes"` in the configuration for your lights - you need to untick this box in the settings and try again!

hello, i just enabled AWS but im still not getting the code, i now gives me an error that says "8/4/2021, 10:15:24 AMappleGoveeWARNING[Window lights] not sending AWS update as not available or enabled at /var/lib/hoobs/apple/node_modules/homebridge-govee/lib/index.js:982:19."

here is the full log:
8/4/2021, 10:13:52 AMapple starting
8/4/2021, 10:13:54 AMappleLoaded plugin 'homebridge-govee'
8/4/2021, 10:13:54 AMappleLoading 1 platforms...
8/4/2021, 10:13:54 AMappleGoveeInitialising plugin v4.0.2 | Node v14.17.4 | HB v1.3.4 | HOOBS v4...
8/4/2021, 10:13:54 AMappleGoveePlugin initialised. Setting up accessories....
8/4/2021, 10:13:54 AMappleBridge is running on port 51826.
8/4/2021, 10:13:55 AMappleGovee[Indoor Thermometer] initialising with options {"logging":"debug","lowBattThreshold":20}.
8/4/2021, 10:13:55 AMappleGovee[Indoor Thermometer] initialised with id [16:10:A4:C1:38:10:F6:1D].
8/4/2021, 10:13:55 AMappleGovee[Door lights] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"yes","enableBT":"no","logging":"debug"}.
8/4/2021, 10:13:55 AMappleGovee[Door lights] initialised with id [3B:BF:A4:C1:38:CE:75:63].
8/4/2021, 10:13:55 AMappleGovee[Window lights] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"yes","enableBT":"no","logging":"debug"}.
8/4/2021, 10:13:55 AMappleGovee[Window lights] initialised with id [29:B6:A4:C1:38:A6:4D:E5].
8/4/2021, 10:13:55 AMappleGovee[Desk lights] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"yes","enableBT":"no","logging":"debug"}.
8/4/2021, 10:13:55 AMappleGovee[Desk lights] initialised with id [3E:57:A4:C1:38:16:0C:73].
8/4/2021, 10:13:55 AMappleGoveeWARNING[Bed lights] is not currently supported, do create a Github issue with the following info:
8/4/2021, 10:13:55 AMappleGoveeWARNING[object Object]
8/4/2021, 10:13:55 AMappleGovee✓ Setup complete. Have a feature request? Visit http://bit.ly/hb-govee-issues to ask!
8/4/2021, 10:13:55 AMappleGovee[noble] stateChange: poweredOn.
8/4/2021, 10:13:55 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:13:55 AMappleGoveeWARNING[Door lights] has been reported [offline].
8/4/2021, 10:13:55 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:13:55 AMappleGovee[Window lights] has been reported [online].
8/4/2021, 10:13:55 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:13:55 AMappleGoveeWARNING[Indoor Thermometer] has been reported [offline].
8/4/2021, 10:13:55 AMappleGovee[Indoor Thermometer] current battery [67%].
8/4/2021, 10:13:55 AMappleGovee[Indoor Thermometer] current temperature [26.1°C].
8/4/2021, 10:13:55 AMappleGovee[Indoor Thermometer] current humidity [41.1%].
8/4/2021, 10:13:55 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:13:55 AMappleGovee[Desk lights] has been reported [online].
8/4/2021, 10:14:10 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:14:10 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:14:10 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:14:10 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:14:25 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:14:25 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:14:25 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:14:25 AMappleGovee[Window lights] current state [on].
8/4/2021, 10:14:25 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:14:40 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:14:40 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:14:40 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:14:40 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:14:55 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:14:55 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:14:55 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:14:55 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:15:10 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:15:10 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:15:10 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:15:10 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:15:24 AMappleGoveeWARNING[Window lights] not sending AWS update as not available or enabled at /var/lib/hoobs/apple/node_modules/homebridge-govee/lib/index.js:982:19.
8/4/2021, 10:15:24 AMappleGoveeWARNING[Window lights] reverting to API as not available or enabled at /var/lib/hoobs/apple/node_modules/homebridge-govee/lib/index.js:1006:17.
8/4/2021, 10:15:24 AMappleGovee[Window lights] [API] sending update {"name":"turn","value":"off"}.
8/4/2021, 10:15:24 AMappleGovee[Window lights] current state [off].
8/4/2021, 10:15:25 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:15:25 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:15:25 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:15:25 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:15:26 AMappleGoveeWARNING[Window lights] not sending AWS update as not available or enabled at /var/lib/hoobs/apple/node_modules/homebridge-govee/lib/index.js:982:19.
8/4/2021, 10:15:26 AMappleGoveeWARNING[Window lights] reverting to API as not available or enabled at /var/lib/hoobs/apple/node_modules/homebridge-govee/lib/index.js:1006:17.
8/4/2021, 10:15:26 AMappleGovee[Window lights] [API] sending update {"name":"turn","value":"on"}.
8/4/2021, 10:15:26 AMappleGovee[Window lights] current state [on].
8/4/2021, 10:15:40 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:15:40 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:15:40 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:15:40 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:15:55 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:15:55 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:15:55 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:15:55 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:16:10 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:16:10 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:16:10 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:16:10 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:16:25 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:16:25 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:16:25 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:16:25 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:16:40 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:16:40 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:16:40 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:16:40 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:16:55 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:16:55 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:16:55 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:16:55 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:17:10 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:17:10 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:17:10 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:17:10 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:17:25 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:17:25 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:17:25 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:17:25 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:17:40 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:17:40 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:17:40 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:17:40 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:17:55 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:17:55 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:17:55 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].
8/4/2021, 10:17:55 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:18:10 AMappleGovee[Door lights] [API] receiving update [{"source":"API","online":"false","powerState":"off","brightness":100,"colorTemInKelvin":9000,"colorTem":9000}].
8/4/2021, 10:18:10 AMappleGovee[Window lights] [API] receiving update [{"source":"API","online":true,"powerState":"on","brightness":15,"color":{"r":0,"b":110,"g":255}}].
8/4/2021, 10:18:10 AMappleGovee[Indoor Thermometer] [HTTP] receiving update [{"source":"HTTP","battery":67,"temperature":2610,"humidity":4110,"online":false}].
8/4/2021, 10:18:10 AMappleGovee[Desk lights] [API] receiving update [{"source":"API","online":true,"powerState":"off","brightness":9,"color":{"r":255,"b":166,"g":205}}].




Thanks, Devin

**Comment by bwp91** [comment]
Which Govee models are they? are they bluetooth only? Could be that they don't support AWS. I believe only wifi or bluetooth+wifi models do...

**Comment by devins-07** [comment]
> Which Govee models are they? are they bluetooth only? Could be that they don't support AWS. I believe only wifi or bluetooth+wifi models do...

thanks for your support, i know i can be challenging,
i have 4 different light strips, the first one which is the one i am trying to get help for is model:H6117 hardware version 1.00.02
the second one is also H6117 hardware version:1.00.01 
and the third one is model H6159 hardware version:1.00.02
and the last one is model:H6126 hardware version:1.00.01 which may not be supported.
Thanks for all you help!, Devin

**Comment by devins-07** [comment]
I do believe that all the models I own have WIFI support except for model H6126

**Comment by bwp91** [comment]
> I do believe that all the models I own have WIFI support except for model H6126

Yes, i've just checked and I agree.

Can you show me your govee plugin config? (hide your username, password and apikey)

**Comment by devins-07** [comment]
(i removed the api key username and password just for the photo, it was there already so that's not the issue)

![1](https://user-images.githubusercontent.com/80927794/128201117-debaa7fc-bdbe-4a5a-8d89-6abe937e20e7.PNG)
![2](https://user-images.githubusercontent.com/80927794/128201133-c05da48e-45de-478c-b2e3-62459353dd02.PNG)
![3](https://user-images.githubusercontent.com/80927794/128201142-8db1f5d2-e8ee-4605-9e6e-79a51e36a0d6.PNG)

**Comment by bwp91** [comment]
From your logs:

```
8/3/2021, 7:13:19 PMappleGovee[Window lights] initialised with id [29:B6:A4:C1:38:A6:4D:E5].
```

In your config you have a totally different device id? It needs to be the above, ie `29:B6:A4:C1:38:A6:4D:E5`

**Comment by devins-07** [comment]
oh, lol I'm dumb. I thought the device id is the MAC address of the govee device. thanks

**Comment by devins-07** [comment]
so now i am getting 
"appleGovee[Window lights] [AWS] message event [GD/3cb99e5aa43fdec9e232b8792242daa5 {"msg":{"transaction":"y_1628431825549053","accountTopic":"GA\/d05aeeec4e8a1c4ab69ff0c3863086eb","type":0,"cmdVersion":1,"cmd":"status"}}]."
is that what i need?

**Comment by bwp91** [comment]
use your phone **with bluetooth OFF** with the govee app to change the scene of your lights, you should then get a message in the log like `the AWS code is on the following line` I cant remember the exact wording

**Comment by devins-07** [comment]
oh thank you, i found it 
"[Window lights] the AWS code for this scene/mode is shown on the next line(s):
owABBQICJgAAAAUAAf//AAAAAIU=,owEC/zIFiwD/AAD//xP//38A/48=,owIAABMB/wAAAAAmAAAAAwAB/5c=,owP/AAAAAAL/MgWLAP8AAP8AAB4=,o/8A////AP//EwH/AAAAAAAAAE4=,MwUE5QAAAAAAAAAAAAAAAAAAANc="
i am going to try continuing from the guide. Also thank you so much for making this plugin and then continuing to help people!!

**Comment by bwp91** [comment]
That’s more like it! Just don’t enter that last quotation mark “ into the plugin if it’s not part of the code!
Good luck and if you get stuck just ask on here…

**Comment by devins-07** [comment]
Thankyou!!!!!

**Comment by devins-07** [comment]
one last thing, say i picked the rainbow scene in the govee and this is what it gave me in the log, [Window lights] the AWS code for this scene/mode is shown on the next line(s):
MwUEFgAAAAAAAAAAAAAAAAAAACQ=

do i put that straight into the configuration so that the config looks like this?
![4](https://user-images.githubusercontent.com/80927794/128206444-7d8bbb9d-1647-4bfc-8e5d-eed1d1d6dd6b.PNG)

**Comment by bwp91** [comment]
Yes that’s it!

**Comment by devins-07** [comment]
weird, when i turn the scene mode on in the eve app nothing happens

**Comment by bwp91** [comment]
Does anything appear in the logs when you click the eve button?

Also, your lights need to be connected to wifi at the time the command is sent, so make sure it’s not connected to your phones bluetooth

**Comment by devins-07** [comment]
my phones bluetooth is off and this is what shows up in the log, 
8/4/2021, 11:18:48 AMappleGovee[Window lights] [AWS] sending update {"msg":{"accountTopic":"GA/d05aeeec4e8a1c4ab69ff0c3863086eb","cmd":"pt","cmdVersion":0,"data":{"value":["MwUEFgAAAAAAAAAAAAAAAAAAACQ="],"op":"mode"},"transaction":"v_1628090328869000","type":1}}.
8/4/2021, 11:18:48 AMappleGovee[Window lights] current scene [Scene].
8/4/2021, 11:18:48 AMappleGovee[Window lights] [AWS] message event [GD/3cb99e5aa43fdec9e232b8792242daa5 {"msg":{"accountTopic":"GA/d05aeeec4e8a1c4ab69ff0c3863086eb","cmd":"pt","cmdVersion":0,"data":{"value":["MwUEFgAAAAAAAAAAAAAAAAAAACQ="],"op":"mode"},"transaction":"v_1628090328869000","type":1}}].
8/4/2021, 11:18:48 AMappleGovee[Window lights] [AWS] receiving update [{"accountTopic":"GA/d05aeeec4e8a1c4ab69ff0c3863086eb","cmd":"pt","cmdVersion":0,"data":{"value":["MwUEFgAAAAAAAAAAAAAAAAAAACQ="],"op":"mode"},"transaction":"v_1628090328869000","type":1,"source":"AWS"}].


also, this is what i looks like in the eve app,

![Screenshot 2021-08-08 at 10 42 58 AM](https://user-images.githubusercontent.com/80927794/128207694-bf25dd84-ce42-409c-b5ee-b17397647960.png)

**Comment by bwp91** [comment]
Can you show me the logs that appear if you change to this scene in the govee app? (if already in that scene then choose another then go back to it)

**Comment by devins-07** [comment]
this is what shows up from the scene change in the govee app

8/4/2021, 11:24:16 AM appleGovee[Window lights] [AWS] message event [GD/3cb99e5aa43fdec9e232b8792242daa5 {"msg":{"transaction":"v_1628434036283925","data":{"value":["MwUEFgAAAAAAAAAAAAAAAAAAACQ="],"opcode":"mode"},"accountTopic":"GA\/d05aeeec4e8a1c4ab69ff0c3863086eb","type":1,"cmdVersion":1,"cmd":"pt"}}].
8/4/2021, 11:24:16 AM appleGovee[Window lights] [AWS] receiving update [{"transaction":"v_1628434036283925","data":{"value":["MwUEFgAAAAAAAAAAAAAAAAAAACQ="],"opcode":"mode"},"accountTopic":"GA/d05aeeec4e8a1c4ab69ff0c3863086eb","type":1,"cmdVersion":1,"cmd":"pt","source":"AWS"}].

**Comment by bwp91** [comment]
Ok, you’re doing nothing wrong, it’s the plugin, you’re code isn’t following the “rules” i thought i had worked out. 

I’ll need to figure a way out of this lol

**Comment by devins-07** [comment]
oh ok, thanks for the reassurance lol and thanks for helping me figure this out.

**Comment by devins-07** [comment]
no rush, do you have an eta though?

**Comment by bwp91** [comment]
looking now

**Comment by bwp91** [comment]
can you update to the beta version of the plugin,

https://github.com/bwp91/homebridge-govee/wiki/Beta-Version#hoobs-4

and verify that when the plugin restarts, in the logs it says something like

```
Govee Bridge Govee Initialising plugin v4.2.1-beta.0 | Node v14.17.4 | HB v1.3.4 | HOOBS v4...
```

once this is done let me know

**Comment by devins-07** [comment]
> can you update to the beta version of the plugin,
> 
> https://github.com/bwp91/homebridge-govee/wiki/Beta-Version#hoobs-4
> 
> and verify that when the plugin restarts, in the logs it says something like
> 
> ```
> Govee Bridge Govee Initialising plugin v4.2.1-beta.0 | Node v14.17.4 | HB v1.3.4 | HOOBS v4...
> ```
> 
> once this is done let me know

hi, do you want me to install the beta version 4.0.3-beta.0?

**Comment by bwp91** [comment]
Should now be 4.2.1-beta.1

**Comment by bwp91** [comment]
the version numbers in hoobs are incorrect

**Comment by devins-07** [comment]
i just did it, this is what it says 
8/4/2021, 5:12:30 PMappleGoveeInitialising plugin v4.2.1-beta.1 | Node v14.17.4 | HB v1.3.4 | HOOBS v4...
8/4/2021, 5:12:30 PMappleGoveePlugin initialised. Setting up accessories....
8/4/2021, 5:12:30 PMappleBridge is running on port 51826.

**Comment by bwp91** [comment]
yea this looks correct. now do the same as before with the govee app to get the scene to appear in the logs, the scene code will be slightly different

**Comment by devins-07** [comment]
this is what it says,
8/4/2021, 5:16:33 PMappleGovee[Window lights] [AWS] message event [GD/3cb99e5aa43fdec9e232b8792242daa5 {"msg":{"transaction":"v_1628543763599241","data":{"value":["MwUEFgAAAAAAAAAAAAAAAAAAACQ="],"opcode":"mode"},"accountTopic":"GA\/d05aeeec4e8a1c4ab69ff0c3863086eb","type":1,"cmdVersion":1,"cmd":"pt"}}].
8/4/2021, 5:16:33 PMappleGovee[Window lights] [AWS] receiving update [{"transaction":"v_1628543763599241","data":{"value":["MwUEFgAAAAAAAAAAAAAAAAAAACQ="],"opcode":"mode"},"accountTopic":"GA/d05aeeec4e8a1c4ab69ff0c3863086eb","type":1,"cmdVersion":1,"cmd":"pt","source":"AWS"}].

**Comment by devins-07** [comment]
i dont know if this helps but its also saying this, 
8/4/2021, 5:17:00 PMappleGovee[Window lights] [AWS] offline event.
8/4/2021, 5:17:00 PMappleGovee[Window lights] [AWS] close event.
i dont know if this helps but

**Comment by devins-07** [comment]
should i remove the scene code from the config and try agian? 
![2000](https://user-images.githubusercontent.com/80927794/128256385-62fb6304-2d6c-418e-bc5d-7a04e3c62d97.PNG)

**Comment by bwp91** [comment]
No, try this as your code:

```
MwUEFgAAAAAAAAAAAAAAAAAAACQ=||pt||opcode
```

**Comment by devins-07** [comment]
i tried it, 
![1234](https://user-images.githubusercontent.com/80927794/128256906-3b3893d9-2d85-4e01-854b-b13af37fb8cc.PNG)
and when i turn the scene mode on in the eve app it doesn't work

**Comment by bwp91** [comment]
can you show logs of the sending command?

**Comment by devins-07** [comment]
what command? the turning the scene mode on in the eve app command?

**Comment by bwp91** [comment]
Yes please!

**Comment by devins-07** [comment]
it says this 
8/4/2021, 5:27:30 PMappleGoveeWARNING[Window lights] not sending AWS update as not connected to AWS at connectionAWS.updateDevice (/var/lib/hoobs/apple/node_modules/homebridge-govee/lib/connection/aws.js:106:13).

**Comment by bwp91** [comment]
Oh, can you try restarting the plugin?

**Comment by devins-07** [comment]
yes

**Comment by devins-07** [comment]
lol sorry, how do i restart the plugin? like restart the raspberrypi?

**Comment by bwp91** [comment]
in the hoobs ui click on Bridges on the left, you should see a govee bridge, and an option to restart the bridge

**Comment by devins-07** [comment]
i just did

**Comment by devins-07** [comment]
in the log, it keeps saying 
8/4/2021, 5:32:02 PMappleGovee[Window lights] [AWS] offline event.
8/4/2021, 5:32:02 PMappleGovee[Window lights] [AWS] close event.

**Comment by devins-07** [comment]
and it still says 
8/4/2021, 5:34:07 PMappleGoveeWARNING[Window lights] not sending AWS update as not connected to AWS at connectionAWS.updateDevice (/var/lib/hoobs/apple/node_modules/homebridge-govee/lib/connection/aws.js:106:13).

**Comment by bwp91** [comment]
i'm not sure why this is happening... this mean for some reason the connection between the plugin and the govee server is being closed.

are you logged into govee in more places than the app and the plugin? can you try changing your govee password?

**Comment by bwp91** [comment]
> and it still says
> 8/4/2021, 5:34:07 PMappleGoveeWARNING[Window lights] not sending AWS update as not connected to AWS at connectionAWS.updateDevice (/var/lib/hoobs/apple/node_modules/homebridge-govee/lib/connection/aws.js:106:13).

yes, sending the command won't work when the connection is closed

**Comment by devins-07** [comment]
yeah ill change my password. do you think that it could be because of alexa being connected to the devices?

**Comment by devins-07** [comment]
also here is the full log file 
[log.txt](https://github.com/bwp91/homebridge-govee/files/6934520/log.txt)

**Comment by bwp91** [comment]
> yeah ill change my password. do you think that it could be because of alexa being connected to the devices?

this could be... i'm not sure tbh

**Comment by devins-07** [comment]
> yeah ill change my password. do you think that it could be because of alexa being connected to the devices?

actually its not alexa because Alexa isn't plugged in right now

**Comment by bwp91** [comment]
but this wasn't happening earlier?

**Comment by devins-07** [comment]
lol no it wasnt happening until i updated to the beta and until i changed the scene code.

**Comment by devins-07** [comment]
i changed my password and it still says the same thing.

**Comment by bwp91** [comment]
any chance your computer raspberry pi could be briefly disconnecting from internet?

aim to send the command from eve when the last status is "connected"

**Comment by devins-07** [comment]
i dont think that its disconnecting unless there is something wrong with the Ethernet port itself, is there a way i can connect to wifi to be sure?

**Comment by devins-07** [comment]
ok, i unplugged Ethernet and plugged it back in, and i think the issue might have been that it was trying to connect over Bluetooth, take a look at the log because I'm not entirely sure.
/4/2021, 9:04:58 PMappleGovee[noble] scanning started.
8/4/2021, 9:04:58 PMappleGovee[Window lights] [AWS] message event [GD/3cb99e5aa43fdec9e232b8792242daa5 {"msg":{"accountTopic":"GA/d05aeeec4e8a1c4ab69ff0c3863086eb","cmd":"turn","cmdVersion":0,"data":{"val":0},"transaction":"v_1628125498592000","type":1}}].
8/4/2021, 9:04:58 PMappleGovee[Window lights] [AWS] receiving update [{"accountTopic":"GA/d05aeeec4e8a1c4ab69ff0c3863086eb","cmd":"turn","cmdVersion":0,"data":{"val":0},"transaction":"v_1628125498592000","type":1,"source":"AWS"}].
8/4/2021, 9:04:58 PMappleGovee[Window lights] current state [off].
8/4/2021, 9:04:58 PMappleGovee[noble] scanning stopped.
8/4/2021, 9:04:58 PMappleGovee[Window lights] has been reported [discoverable] via BLE.
8/4/2021, 9:04:58 PMappleGovee[Window lights] has been reported [connected] via BLE.
8/4/2021, 9:04:59 PMappleGovee[Window lights] [BLE] sending update {1: 0}.
8/4/2021, 9:04:59 PMappleGovee[Window lights] current state [off].

**Comment by bwp91** [comment]
Yes, the aws connection looks to be working again.

When turning the device on and off, the plugin is using bluetooth to do this.

Have you tried the scene button in Eve again? this will use the aws connection not bluetooth.

**Comment by devins-07** [comment]
its disconnecting and reconnecting again, im just going to make a fresh install of hoobs.

**Comment by bwp91** [comment]
> its disconnecting and reconnecting again, im just going to make a fresh install of hoobs.

Any luck? Tbh I can’t see how this would help. But maybe this disconnect/reconnect has improved in general?
